src/components/SignUp/SignUp.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './SignUp.css';
+import { Link } from 'react-router-dom';
+
+export default function SignUp() {
+  return (
+    <>
+    <div className="login-container">
+        <h3 className='login-container-heading'>Account Information</h3>
+        <div className="login-card">
+            <h2>Sign Up</h2>
+            <form className="logIn-form">
+                <label htmlFor="full-name" aria-required="true">Full Name<span>*</span></label>
+                <input type="text" name="ful-name" id="full-name"/>
+
```

Comment:
changes: in name there is a typo 'ful-name'

Reply:
thanks, will correct it.